src/App.jsx

```diff
@@ -14,70 +16,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const firstSelected = 'Jam';
+  const [value, setValue] = useState(firstSelected);
```

Comment:
```suggestion
  const [selectedGood, setSelectedGood] = useState(goods[8]);
```
Try to provide more meaningful names for variables. There is also no need to create a variable for Jam.
You can get this value from goods array